POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Services/ActivationService.cs

```diff
@@ -21,6 +25,88 @@ internal class ActivationService
 
         private object _lastActivationArgs;
 
+        private void newNotification()
+        {
+            
+            // In a real app, these would be initialized with actual data
+            string title = "Power Toys";
+            string content = "Power Toys is installed";
+            string image = "../Assets/Logo.png";
+            string logo = "../Assets/MiniLogo.png";
+
+            // Construct the visuals of the toast
+            ToastVisual visual = new ToastVisual()
+            {
+                BindingGeneric = new ToastBindingGeneric()
+                {
+                    Children =
+                    {
+                        new AdaptiveText() {
+                            Text = title
+                        },
+
+                        new AdaptiveText()
+                        {
+                            Text = content
+                        },
+
+                        new AdaptiveImage()
+                        {
+                            Source = image
+                        }
+                    },
+
+                    AppLogoOverride = new ToastGenericAppLogo()
+                    {
+                        Source = logo,
+                        HintCrop = ToastGenericAppLogoCrop.Circle
+                    }
+                }
+            };
+
+            // In a real app, these would be initialized with actual data
+            int conversationId = 384928;
```

Comment:
Is this a reserved constant tied to an AppID?

Reply:
Removed this component, it was a part of the boilerplate that I forgot to remove